script/1_0_2current_en_script/Arcueid/Day 1/01_00_ARC01_1.txt

```diff
@@ -3,19 +3,19 @@ The rail line above ground switches to an underground line without a hint of war
 The train, spewing artificial light, moves through the darkness as if it were diving into it.
 <Page2>
 The sound of creaking vehicles.
-Artificial lights passing by at even intervals.
+Artificial lights passing by at regular intervals.
 As the tremors of the seat overlap with the second hand of the clock, I think about the passage of time and space.
 <Page3>
 It's early morning, 6:33 am.
 Half an hour has passed since I got on this train.
-That's how much time and distance has passed since I left the homes of the people who have taken care of me over the years and to whom I will never return.
-How I accept this may be a measure of the depth of my compassion as a human being.
+I can't go back anymore, after I left the family that took care of me all these years, all this time and distance is now keeping us apart.
+How to accept that might come to the measurement of compassion as a human being.
```

Comment:
Hmm, this one doesn't quite make sense to me.

Reply:
Is the idea behind 'come to' as a word choice here that it is by analyzing how he accepts it that he will come to the conclusion of how compassionate he is? If that's the intent, "In thinking about how I accept that I might come to the measure of my compassion as a human being", but that still feels a little stilted. I would probably choose the original sentence here.